weather-app/main.js

```diff
@@ -0,0 +1,7 @@
+const {getWeather,getGeoData}=require('./helper')
+
+const LOCATION = 'Grodno'
```

Comment:
try to pass empty location 
```js 
const location = ''
``` 
and see what happens. We should return our error message not the Error object